README.md

```diff
@@ -62,13 +63,16 @@ You can then use the following selector anywhere in your project:
 | borderRadius | string | ```34px``` | Sets the track border radius |
 
 ### events
-| Event Name | Returns | Description |
-| --- | ---  | --- |
-| **setIsActive** | `isActive` | Clicking the toggle button emits an its current `isActive` boolean state |
+All standard input events (focus, input, change, etc.) as you could put on any other standard input.
 
 Listening to the event e.g:
 ```html
-<vue-toggle-btn @isActive="myLocalSetterFunction($event)"></vue-toggle-btn>
+<vue-toggle-btn @input="myLocalSetterFunction($event)"></vue-toggle-btn>
+```
+
+The component can also work with v-model e.g:
```

Comment:
Maybe it could be benefitial for people if you can add some more text in the comment what is a common use case for usage of v-model